content/en/docs/reference/command-line-tools-reference/feature-gates/ServiceAccountNodeAudienceRestriction.md

```diff
@@ -0,0 +1,14 @@
+---
+title: ServiceAccountNodeAudienceRestriction
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: beta
+    defaultValue: true
```

Comment:
```suggestion
    defaultValue: false
```

Reply:
The default value for beta `ServiceAccountNodeAudienceRestriction` feature gate is `false` in k8s v1.32

Reply:
Please take a look at the corresponding upstream code:
https://github.com/kubernetes/kubernetes/blob/master/pkg/features/versioned_kube_features.go#L706C2-L706C39

In k8s v1.33 it will be true.